lib/backend/Memory.js

```diff
@@ -0,0 +1,190 @@
+import map from 'async/map';
+
+/**
+* Pipeline - executes multiple commands sent as a batch
+*/
+class Pipeline {
+    constructor(cmds, db) {
+        this.cmds = cmds;
+        this.db = db;
+    }
+
+    /**
+    * @param {callback} cb - callback
+    * @return {undefined}
+    */
+    exec(cb) {
+        process.nextTick(() => {
+            map(this.cmds, (item, next) => {
```

Comment:
I have added JsDoc to the constructor, which defines what cmds is and what a typical example looks like.